scripts/xe-reset-networking

```diff
@@ -52,7 +52,7 @@ def write_inventory(inventory):
 def valid_vlan(vlan):
 	if not re.match('^\d+$', vlan):
 		return False
-	if int(vlan)<1 or int(vlan)>=4095:
+	if int(vlan)<0 or int(vlan)>=4095:
```

Comment:
4095 is not allowed to create a VLAN. But it would be better to keep it `vlan>4094`.
https://github.com/xapi-project/xen-api/blob/master/ocaml/xapi/xapi_vlan.ml#L53